website/static/data/events.json

```diff
@@ -1,4 +1,16 @@
 [
+  {"title": "Weekly Report｜10.15-10.31 Feature Highlights Update in Progress",
```

Comment:
Okay, so now as the titles have gotten longer, hierarchical difference between event title and date can't be made. So I suggest we assign `event-card-time` as different color (this maybe - #6a6a6a) and increase top (to 55% will do)

Reply:
good idea, I will merge this PR first, then would you please do some changes then update it?

Reply:
Sure